### DISCUSSION TITLE
cursor in popup window is not synced with selected item

### DISCUSSION THREAD

**Discussion by spamwax** [neoclide/coc.nvim#4352] [Q&A] [answered]
When cycling through the items in a popup window, sometimes the selected item is not the same as the highlighted item in the menu!

https://user-images.githubusercontent.com/1251233/200987956-7963d9db-a6b7-4869-a725-da4bc3b4ae03.mov

I have this settings: `"suggest.noselect": true,`

**Comment by chemzqm** [comment]
Can't reproduce, what's your vim version?

**Comment by spamwax** [comment]
```
▶︎ nvim --version
NVIM v0.9.0-dev-284+g0faf007a2
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by hamid@macpro7.local
```

```
▶︎ node --version
v19.0.1
```

coc.nvim
```
commit d54d52b950df0ee1c76813a4fa8b06c5b87714f7 (HEAD -> release, origin/release)
Author: Qiming Zhao <chemzqm@gmail.com>
Date:   Tue Nov 8 23:14:10 2022 +0800

    commit a7c0900fa178e36f010c6ccc4e02cb3f8090a6c9
```

**Reply by spamwax** [reply]
I create a [bug report](https://github.com/neoclide/coc.nvim/issues/4357) with `mini.vim`

**Comment by spamwax** [comment]
It appears to be related to the `"suggest.noselect": true,` option.
If you see this new video, when the cursor in popup window is not active and I press `ctrl-N` it highlights the first item but show the second item in nvim. 

https://user-images.githubusercontent.com/1251233/201307078-8e747c03-d65a-48af-975e-603d5b9ad82b.mov

I am guessing, somewhere the code doesn't take into account that `"suggest.noselect": true,` is active and it thinks the first item is selected. So when I press `ctrl-N` it moves the highlight cursor but shows the 2nd item since it already thinks the 1st time was displayed.

**Comment by chemzqm** [comment, marked as answer]
Wrong lable, fixed on latest release branch of coc.nvim.
